src/main/java/org/team114/lib/pathgenerator/ParametricOneVariableSpline.java

```diff
@@ -6,12 +6,12 @@
      * @param t The parameter from 0 to 1 inclusive, representing the percent through the spline.
      * @return The value of the spline.
      */
-    public double getValueAt(double t);
+    double at(double t);
```

Comment:
I don't know if conventions have evolved yet, but interfaces allow private methods in Java 9, so the explicit/implicit thing applies.

Reply:
The public is optional but I personally prefer it for readability and I don't think changing getValueAt to at is very beneficial. Changing it to eval though to match Polynomial may be an option.

Reply:
Public, okay. Intelij doesn't like it, but that's survivable. GetValueAt is too verbose. At means literally the same thing. If eval is really more appropriate, it's reasonable too.

Reply:
Oh, and could we rename this interface to something shorter?

Reply:
I like the name as is, and I can't really think of a name that doesn't really butcher the meaning/readability. The only thing I can think of is ParametricOneDSpline.

Reply:
ParametricMonoSpline?

Reply:
Kind of makes it sound like a new type of spline.

Reply:
Is there anything else specific it could reasonably mean?